app/src/components/widgets/charts/failedCasesTrendChart/configuration.js

```diff
@@ -0,0 +1,49 @@
+const range = (start = 0, stop, step = 1) => {
+  if (arguments.length <= 1) {
+    stop = start; // eslint-disable-line
+  }
+
+  const length = Math.max(Math.ceil((stop - start) / step), 0);
+  let idx = 0;
+  const arr = new Array(length);
+
+  while (idx < length) {
+    arr[idx++] = start; // eslint-disable-line
+    start += step; // eslint-disable-line
+  }
+
+  return arr;
+};
+const rangeMaxValue = (itemsLength) => (itemsLength > 6 ? Math.round(itemsLength / 12) : 1);
```

Comment:
What is 6? What is 12?
Move magic numbers to constants with meaningful names.

Reply:
@DiscoElevator needs further investigation, code was moved from: https://github.com/reportportal/service-ui/blob/master/src/main/resources/public/js/src/newWidgets/_C3ChartWidgetView.js#L66